### DISCUSSION TITLE
HOW to install mkcert

### DISCUSSION THREAD

**Discussion by touriya34** [FiloSottile/mkcert#562] [Ideas]
HOW to install mkcert in windows

**Comment by jdreesen** [comment]
https://github.com/FiloSottile/mkcert#windows

**Reply by wnunezc** [reply]
hello, i need it but silently its posible?

check this post: https://stackoverflow.com/questions/78011051/runs-in-powershell-mkcert-install-silently-to-avoid-any-dialog

To do an unattended installation from a C# application, run:

`$ mkcert -install`

It shows me this annoying warning:

![image](https://github.com/FiloSottile/mkcert/assets/1207306/fff44ef5-fe58-44a9-b9fb-c1a04f7a2605)
